### PR TITLE
feat(components) : rework on the list style

### PR DESCRIPTION
**Pull request by talend-sromain**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

The list has some display

**What is the new behavior?**

Some aspects has changed :
- uppercase on titles removed
- some paddings has changed
- colors on titles & subtitles

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:
Reference: http://guidelines.talend.com/document/92132#/navigation-layout/list-br

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://249.talend.surge.sh)

File: packages/components/src/List/DisplayLarge/DisplayLarge.scss [hunk 1]
```diff
@@ -35,19 +36,34 @@ $tc-list-display-large-background-color: lighten($tc-list-display-large-color, 2
...
-	margin-top: $padding-normal;
+	margin-top: $padding-smaller;
 	display: flex;
 
-	& ul {
+	ul {
+		// sass-lint:disable-block no-vendor-prefixes
 		flex: 1;
+		list-style: none;
+		-webkit-padding-start: 15px;
```
Comment (frassinier) [inline packages/components/src/List/DisplayLarge/DisplayLarge.scss]: this does not work with Internet Explorer and should be `<dl>`

File: packages/components/src/List/DisplayLarge/DisplayLarge.component.js [hunk 2]
```diff
@@ -79,14 +81,14 @@ function rowRenderer({ id, index, columns, item, itemProps, titleProps }) {
 			</div>
 			<div className={theme.columns}>
 				{info.map((group, i) => (
-					<ul key={i}>
+					<dl key={i}>
 						{group.map((obj, j) => (
-							<li key={j}>
+							<dd key={j}>
 								<span className={theme.label}>{obj.label}</span>
 								<span className={theme.value}>{obj.value}</span>
```
Comment (frassinier) [inline packages/components/src/List/DisplayLarge/DisplayLarge.component.js]: In fact, should be:

```jsx
<div className={theme.columns}>
  {info.map((group, i) => (
    <dl key={i}>
        group.reduce((acc, obj, j) => acc.concat([
            this.renderObjKey(obj, j),
            this.renderObjValue(obj, j),
        ]), [])
    </dl>
</div>
```

And

```javascript
renderObjKey(obj, j) {
    return <dt key={j} className={theme.label}>{obj.label}</dt>
}

renderObjValue(obj, j) {
    return <dd key={j} className={theme.value}>{obj.value}</dd>
}
```

Comment (jmfrancois) [inline packages/components/src/List/DisplayLarge/DisplayLarge.component.js]: I'm not sure this is more readable, and if you do that renderObjKey is a React pure component, so you also have to add proptypes, tests, ...

**Pull request closed without merging**